Features/front end

Rifqihamza opened the pull request:
- change style filters in statistic page
- change style filters in export page

Commit:
change color pagination

iwansal64 committed:
[fix](ReportTable) Truncate for all report table contents

iwansal64 committed:
[feat](API) Add RateLimiter

iwansal64 committed:
[improve](ReportTable) Now, PIC dropdown has search filter in it

iwansal64 committed:
[improve](ReportTable) More informative label in Detail modal

Commit:
Merge remote-tracking branch 'reportRemote/pre-stable' into features/FrontEnd

iwansal64 committed:
[improve](ExportPage) Change the style of selections

Commit:
Merge branch 'design/export-page' into features/FrontEnd

Commit:
changes style StatisticsPage on filters options

iwansal64 reviewed (commented):
Harus nya ke pre-stable ga si bangg?

Commit:
change style of the welcome page and sidebar admin

iwansal64 committed:
[fix](RowOptions) Fix the wrong height of the row options

Commit:
update for Dashboard Pages
- Update SidebarAdmin to include new links and styles
- Update ReportDesktopTable to include new columns and styles
- Update ReportMobileCard to include new styles
- Update ReportMobileTable to include new styles
- Update ReportDetailModal to include new styles and structure
- Remove old ReportMobileCard component
- Update ReportTablePage to include new styles and structure
- Update ReportTableToolbar to include new styles and structure
- Update SelectFilterComponent to include new styles and structure
- Update WelcomePage to include new styles and structure
- Update _SideBarToggle to include new styles and structure
- Update index.astro to include new styles and structure
- Update global.css to include new styles and structure

iwansal64 committed:
[fix](AdminDashboard) text color
